eth-balance/src/lib.rs

```diff
@@ -1,33 +1,50 @@
 #[rustfmt::skip]
 pub mod pb;
 
-use num_bigint;
-use substreams::scalar::BigInt;
-use substreams::store::{StoreNew, StoreSet, StoreSetRaw};
-use substreams::{store, Hex};
+use substreams_helper::pb::evm_token::v1 as token;
+// use pb::evm_token::v1 as token;
+use substreams::Hex;
 use substreams_ethereum::pb::eth as pbeth;
+use substreams_helper::token as token_helper;
+
+#[substreams::handlers::map]
+fn map_balances(block: pbeth::v2::Block) -> Result<token::Transfers, substreams::errors::Error> {
+    let mut transfers = vec![];
 
-#[substreams::handlers::store]
-fn store_balance(block: pbeth::v2::Block, output: store::StoreSetRaw) {
     for transaction in &block.transaction_traces {
+        let mut balance_changes = vec![];
         for call in &transaction.calls {
             for balance_change in &call.balance_changes {
-                // TODO: replace this with substreams::scalar::BigInt once the wrapper is integrated
-                let new_value = balance_change
-                    .new_value
-                    .as_ref()
-                    .map(|value| {
-                        num_bigint::BigInt::from_bytes_be(num_bigint::Sign::Plus, &value.bytes)
-                            .into()
-                    })
-                    .unwrap_or(BigInt::zero());
-
-                output.set(
-                    transaction.end_ordinal,
-                    format!("Address:{}", Hex(&balance_change.address).to_string()),
-                    &new_value.to_string(),
-                )
+                balance_changes.push(token::TokenBalance {
+                    log_ordinal: balance_change.ordinal,
+                    token: token_helper::get_eth_token(),
+                    address: Hex(&balance_change.address).to_string(),
+                    old_balance: token_helper::bigint_to_string(balance_change.old_value.clone()),
```

Comment:
Impl Display trait for bigint and use instead. We can have the impl in substreams helper also